agb/src/display/tiled/vram_manager.rs

```diff
@@ -289,23 +333,26 @@ impl VRamManager {
 
     pub(crate) fn gc(&mut self) {
         for tile_index in self.indices_to_gc.drain(..) {
-            let index = tile_index.index() as usize;
-            if self.reference_counts[index].current_count() > 0 {
+            let key = tile_index.refcount_key() as usize;
+            if self.reference_counts[key].current_count() > 0 {
                 continue; // it has since been added back
             }
 
             let tile_reference = Self::reference_from_index(tile_index);
             unsafe {
-                TILE_ALLOCATOR.dealloc_no_normalise(tile_reference.0.cast().as_ptr(), TILE_LAYOUT);
+                TILE_ALLOCATOR.dealloc_no_normalise(
```

Comment:
I wonder how common having different tile sizes is going to be :thinking:. The no_normalise saves quite a lot of CPU time, but can case pretty bad fragmentation if you use lots of tile formats